Automattic/wp-calypso
# Add an agency bluehost plan

- Fixes: Automattic/martech#2590
## Proposed Changes

* This change rapidly ships to production, under a feature flag a, the agency bluehost plan dirty mockup
* There are some changes here that are temporary, and are not required initially until final copy is completed.

<img width="1532" alt="image" src="https://github.com/Automattic/wp-calypso/assets/3422709/5111d5bd-ec21-4f22-b5bd-cdc0ba586dcc">


## Testing Instructions

* Go to step `/start/plan?flags=bluehost-in-plans-grid` and `/plans?flags=bluehost-in-plans-grid`
* Make sure the agency Bluehost plan is visible
* Also make sure everything works well with no breakage, when viewed above pages without the flag

## Pre-merge Checklist
